src/input/Input.js

```diff
@@ -138,15 +139,15 @@ class Input extends React.Component {
           )}
         </Animated.View>
 
-        {!!errorMessage && (
+        {renderErrorMessage && (
           <Text
             {...errorProps}
             style={StyleSheet.flatten([
               styles.error(theme),
               errorStyle && errorStyle,
             ])}
           >
-            {errorMessage}
+            {errorMessage || ''}
```

Comment:
```suggestion
            {errorMessage}
```

Reply:
@iRoachie but In all SearchBar renderErrorMessage is already false ?